recipes/zulu-openjdk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile
+from six import StringIO
+
+
+class TestPackage(ConanFile):
+
+    def build(self):
+        pass # nothing to build, but tests should not warn
```

Comment:
Maybe add a FIXME about needing to test the cmake integration + adding a test/example with jni.

Reply:
I agree that tests can always be improved, and a note to add a test project that actually links agains jni required libs would be nice, I know that,  but if there is not other required fix I will not re request reviews because of this now.